src/aiida/orm/nodes/data/structure.py

```diff
@@ -1876,6 +1885,35 @@ def _get_object_pymatgen_molecule(self, **kwargs):
         positions = [list(site.position) for site in self.sites]
         return Molecule(species, positions)
 
+    def to_atomistic(self):
+        """
+        Returns the atomistic StructureData version of the orm.StructureData one.
+        """
+        if not has_atomistic():
+            raise ImportError(
+                'aiida-atomistic plugin is not installed, \
+                please install it to have full support for atomistic structures'
+            )
+        else:
+            from aiida_atomistic import StructureData as AtomisticStructureData
+            from aiida_atomistic import StructureDataMutable as AtomisticStructureDataMutable
```

Comment:
Why use alias? Just at the changes here I think `StructureData` and `StructureDataMutable` are clear.

Reply:
yep, will change it, thanks!